libvast/vast/logger.hpp

```diff
@@ -15,189 +15,69 @@
 
 #include "vast/config.hpp"
 #include "vast/detail/discard.hpp"
-#include "vast/detail/pp.hpp"
-#include "vast/detail/type_traits.hpp"
-
-#include <caf/detail/pretty_type_name.hpp>
-#include <caf/logger.hpp>
+#include "vast/error.hpp"
+
+// VAST_INFO -> spdlog::info
+// VAST_VERBOSE -> spdlog::debug
+// VAST_DEBUG -> spdlog::trace
```

Comment:
See other comment on this: The current implementation has debug -> debug mapping.

Reply:
The comment is correct, I updated the implementation.